Python/pywarpx/_libwarpx.py

```diff
@@ -1110,6 +1110,8 @@ def get_mesh_current_density_fp_pml(level, direction, include_ghosts=True):
         return _get_mesh_field_list(libwarpx.warpx_getCurrentDensityFP_PML, level, direction, include_ghosts)
     except ValueError:
         raise Exception('PML not initialized')
+
+
```

Comment:
For all the changes to this file: I understand the desire for neatness but since this file is present in the upstream repo these changes only serve to diverge our fork from upstream without adding any functionality. You can submit an upstream PR for these changes if you like but we shouldn't have this be changed only in our repo.